packages/sakuli-legacy/src/context/sahi/fetch/multiple-element-api.interface.ts

```diff
@@ -4,12 +4,56 @@ import { SahiRelation } from "../relations/sahi-relation.interface";
 import { WebElement } from "selenium-webdriver";
 
 export interface MultipleElementApi {
+  /**
+   * Collects all specified elements
+   *
+   * @example
+   * Assuming the following HTML-snippet:
+   * ```html
+   *     <div class="city">Los Angeles</div>
+   *     <div class="city>Berlin</div>
+   * ```
+   *
+   * Highlights the div containing second city:
+   * ```typescript
+   *     const cities = await _collect('_div', 'city');
+   *
+   *     await _highlight(cities[0]);
```

Comment:
```suggestion
   *     await _highlight(cities[1]);
```
Shouldn't it be `cities[1]` for the second city? :thinking:

Reply:
You are absolutely right :joy: